### PR TITLE
Create account button on Login page

### PR DESCRIPTION
**Pull request by heyitsarthur**
Added create account button on Login page.

Login page -> RegistrationPage

### CODE REVIEW

**Comment by heyitsarthur** [comment]
Screenshot 2023-02-27 165616

**Review by clarag02 — COMMENTED** [review]
Looks good!

**Review by clarag02 — APPROVED** [review]

File: Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -35,6 +35,13 @@ const LoginPage = () => {
                 >
                     Login
                 </Button> 
+                <Button 
+                    className='register-btn' 
+                    onClick={handleLoginBtnClick}
```
Comment (DanielC2001) [inline Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx:40]: Okay, so the _handleLoginBtnClick_ method handles making an API call to log the user in. This method shouldn't be here, 

Instead make a new handler for it. Name it something like _handleRegistrationRedirect_. 

Take a look at the hooks file for the LoginPage. If you look at the _handleLoginBtnClick_ method you'll see there's a variable called navigate that's used for redirection.

EDIT 2: Also remove the href prop, the navigate variable in the hooks file handles all the redirection so you don't have to!

**Review by DanielC2001 — CHANGES_REQUESTED** [review]

File: Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx (hunk 1 above)
Comment (heyitsarthur) [reply Frontend/linkedout/src/main/pages/LoginPage/LoginPage.tsx:40]: Thank you for these suggestions! I will implement them and notify you when it's done.

**Review by DanielC2001 — APPROVED** [review]

**Pull request merged**